cypress/support/PersistentLogs.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+ */
+
+var testCaseRegistry = [];
+var indexRegistry = 0;
+var logEntries = '';
+
+// Override the original console.log() function
+var originalConsolLog = console.log;
```

Comment:
I'd recommend "const" over "var" to avoid accidental hijacking elsewhere in the code

Reply:
Done.

Reply:
Is this pushed? There's a bunch of things that still use `var`.